camera.js

```diff
@@ -152,7 +152,6 @@ function brainLoaded() {
 }
 
 function classifyPose() {
-  // console.log('Classify pose');
```

Comment:
probably keep all the console logs for now

Reply:
done